### PR TITLE
feat: add timeline component and capitalize util

### LINKED ISSUE #10: author page: timeline

### PR DESCRIPTION
**Pull request by spirithntr**

### CODE REVIEW

File: src/components/Timeline/Timeline.jsx [hunk 1]
```diff
@@ -0,0 +1,47 @@
+import React from 'react'
+import PropTypes, { string } from 'prop-types'
+import {
+  Timeline as TimelineComponent,
+  TimelineItem
+} from 'vertical-timeline-component-for-react'
+import { capitalize } from '../../utils/utils'
```
Comment (rss-maksim) [inline src/components/Timeline/Timeline.jsx]: You can name utils.js file as index and will be able to import as ```import { capitalize } from '../../utils'```

File: src/components/Timeline/Timeline.jsx [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+
+// TODO: change colors according to style guide
+const primaryColor = '#243665'
```
Comment (rss-maksim) [inline src/components/Timeline/Timeline.jsx]: You can put new colors in common theme and get them from theme. palette...

File: src/components/Timeline/Timeline.jsx [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+const complementaryColor = '#8BD8BD'
+const filler = '#ddd'
+const Timeline = ({ biography }) => (
+  <TimelineComponent lineColor={filler}>
+    {biography.map((event, index) => (
+      <TimelineItem
+        key={index}
+        dateText={event.date}
+        style={{ color: complementaryColor }}
```
Comment (rss-maksim) [inline src/components/Timeline/Timeline.jsx]: It's better to use const for ```{ color: complementaryColor }```

File: src/components/Timeline/Timeline.jsx [hunk 4]
```diff
@@ -0,0 +1,47 @@
...
+        dateInnerStyle={{
```
Comment (rss-maksim) [inline src/components/Timeline/Timeline.jsx]: It's better to use const and put it in the start of file

File: src/components/Timeline/Timeline.jsx [hunk 5]
```diff
@@ -0,0 +1,47 @@
...
+          background: primaryColor,
+          color: complementaryColor,
+          'padding-left': '20px'
+        }}
+        bodyContainerStyle={{
```
Comment (rss-maksim) [inline src/components/Timeline/Timeline.jsx]: t's better to use const and put it in the start of file OR event use css

File: src/components/Timeline/Timeline.jsx [hunk 6]
```diff
@@ -0,0 +1,47 @@
+import React from 'react'
+import PropTypes, { string } from 'prop-types'
+import {
+  Timeline as TimelineComponent,
+  TimelineItem
+} from 'vertical-timeline-component-for-react'
+import { capitalize } from '../../utils/utils'
+
+// TODO: change colors according to style guide
```
Comment (vadzim-sasonkin) [inline src/components/Timeline/Timeline.jsx]: You can take `useTheme `hook  and get material theme with pallette and colors to your component

File: package-lock.json [hunk 7]
```diff
@@ -14122,6 +14127,14 @@
         }
       }
     },
+    "react-visibility-sensor": {
```
Comment (vadzim-sasonkin) [inline package-lock.json:19475]: I did not find usage of 
`react-visibility-sensor
`and
`classnames`
Do we plan to add them in the future? 
It's not important for approving, just for information

**Review by vadzim-sasonkin — DISMISSED** [review]
For request change, I mean using the theme colors

**Review by rss-maksim — APPROVED** [review]

**Pull request merged**